guidelines.md

```diff
@@ -0,0 +1,30 @@
+# Meetup Quality Guidelines For DevOps Malmö
+
+**Created at:** January 2018
+   (Updated October 2018, October 2019)
+
```

Comment:
I think it's better to add `Created by:` and add location `like Malmö  Sweden` is it's in Document :thinking:

Reply:
Moving the document to text file makes it easy to check who did what in git. So I would actually want to remove the "Created by" and "Created at".